app/controllers/pxe_controller/pxe_customization_templates.rb

```diff
@@ -186,7 +184,7 @@ def template_set_record_vars(ct)
     ct.name = @edit[:new][:name]
     ct.description = @edit[:new][:description]
     ct.type = @edit[:new][:typ]
-    ct.pxe_image_type = PxeImageType.find_by_id(@edit[:new][:img_type])
+    ct.pxe_image_type = PxeImageType.find(@edit[:new][:img_type])
```

Comment:
is it safe to replace thid `find_by_id` with `find`?

Reply:
I do have `CustomizationTemplate` records in my devel database, that have `pxe_image_type_id` set to null. Whether or not that is a bug -- I'm not sure (I do find that odd though).

Reply:
In another words, it seems that I have to revert the above line.

Reply:
Reverted.